WardReports/Generate_weekly_ward_report_scripts.py

```diff
@@ -0,0 +1,186 @@
+import pandas as pd
+import numpy as np
+
+import datetime as dt
+import math
+
+import civis
+import os
+
+#########################################################
+#Pulling in data for email (to be transferred to a different script later)
+
+client = civis.APIClient()
+
+#actual ward table import to be used once testing is complete
+'''
+ward_email_data = civis.io.read_civis(database='City of Chicago',
+                                      table = 'scratch.ward_office_info',
+                                      use_pandas = True)
+                                      '''
+
+#Generate fake table that should be replaced with actual ward table later
+wards = list(range(1,51))
+emails = ['srao@civisanalytics.com' for i in range(50)]
+platform_user = ['Yes' for i in range(25)] + ['No' for i in range(25)]
+ward_email_data = pd.DataFrame(list(zip(wards, emails, platform_user)),
+               columns =['WARD', 'Ward_Office_Email', 'Platform User'])
+
+#Number of weeks since the census started
+def weeks_of_census():
+    census_start = dt.date(2020, 3, 12)
+    today_date = dt.date.today()
+    delta = today_date - census_start
+    num_days = delta.days
+    weeks = math.floor(num_days/7)
+    return weeks
+
+#Pull ward aggregation and household data
+query = """SELECT * FROM cic.ward_visualization_table;"""
+ward_agg = civis.io.read_civis_sql(query,database='City of Chicago',use_pandas = True)
+ward_agg.dropna(inplace=True)
+
+#Calculate counted and uncounted households per ward
+ward_agg['Counted Households'] = round(ward_agg['mail_return_rate_cen_2010'] * (ward_agg['tot_housing_units_acs_13_17']/100))
+ward_agg['Uncounted Households'] = ward_agg['tot_housing_units_acs_13_17'] - ward_agg['Counted Households']
+ward_agg['Percent Counted'] = round(ward_agg['Counted Households']*100/ward_agg['tot_housing_units_acs_13_17'],1)
+ward_agg['Percent Uncounted'] = round(ward_agg['Uncounted Households']*100/ward_agg['tot_housing_units_acs_13_17'],1)
+
+#Calculate City of Chicago uncounted stats
+total_reported_perc = round(ward_agg['Counted Households'].sum()*100/ward_agg['tot_housing_units_acs_13_17'].sum(),1)
+households_left = ward_agg['Uncounted Households'].sum()
+
+best_performer = int(ward_agg[ward_agg['Percent Counted']==ward_agg['Percent Counted'].max()]['ward'].values[0])
```

Comment:
we dont necessarily need to change this now, but this might need to change if we decided to calculate best performer by a relative measure to the Civis HTC score, which i think is pretty likely